SoraUI/Classes/EmptyState/EmptyState+ViewController.swift

```diff
@@ -70,7 +70,7 @@ public extension EmptyStateViewOwnerProtocol where Self: UIViewController {
 
         let newEmptyStateView = emptyStateDataSource.viewForEmptyState ?? createDefaultEmptyStateView()
         let contentView = contentViewForEmptyState
-        contentView.insertSubview(newEmptyStateView, at: 0)
```

Comment:
Why do we need to replace this?

Reply:
`newEmptyStateView` is placed under UITableView, which handles all the touches, so no "button pressed" event passes to `newEmptyStateView`